packages/frontend/package.json

```diff
@@ -80,6 +80,7 @@
     "babel-eslint": "^10.1.0",
     "babel-jest": "^27.0.1",
     "babel-loader": "^8.1.0",
+    "concurrently": "^6.5.1",
```

Comment:
🔪 Was this accidental? It doesn't appear to be used in this PR

Reply:
The prepush git hook script in `package.json` makes use of it. Hence, my adding it as a dev dependency.